### DISCUSSION TITLE
CraftCMS one-click install

### DISCUSSION THREAD

**Discussion by mbeaudru** [coollabsio/coolify#2648] [Service Template Requests]
The most popular modern alternative to Wordpress is CraftCMS (https://craftcms.com) and I think it would be awesome to have it in the one-click install list 👍

**Comment by mbeaudru** [comment]
Related: https://github.com/coollabsio/coolify/discussions/2047

**Comment by Thijmen** [comment]
I love CraftCMS, however it is not a great idea to deploy a development environment to a production environment. CraftCMS needs developing first, locally, in order to go to production. It's in the best practices guide of CraftCMS.

It then can be deployed as a Nixpack application if I recall correctly.

**Comment by espensgr** [comment]
Craft themself has [docker images](https://github.com/craftcms/docker) for production and dev, but as @Thijmen says there is not much point in a one-click install.
Would be nice to have a resource article under [application](https://coolify.io/docs/resources/applications/index) in the docs maybe for this?
